### PR TITLE
fix issue 3557

### PR DESCRIPTION
**Pull request by ritch**
Resolves https://github.com/voxel51/fiftyone/issues/3557

This is tested with this [new example operator](https://github.com/voxel51/fiftyone-plugins/pull/58/files).

### CODE REVIEW

**Review by imanjra — APPROVED** [review]
LGTM

File: app/packages/operators/src/state.ts [hunk 1]
```diff
@@ -123,6 +130,7 @@ const useExecutionContext = (operatorName, hooks = {}) => {
         extended,
         filters,
         selectedSamples,
+        selectedLabels,
       },
       hooks
     );
```
Comment (msamadony) [inline app/packages/operators/src/state.ts:136]: (Github doesn't allow me to comment on the actual line below)

Shouldn't the line below (the second argument of `useMemo`) also be adapted to include `selectedLabels` as well?

i.e. change it from:
```
}, [params, datasetName, view, extended, filters, selectedSamples, hooks]);
```

to
```
}, [params, datasetName, view, extended, filters, selectedSamples, selectedLabels, hooks]);
```

Comment (ritch) [reply app/packages/operators/src/state.ts:136]: I’ll make this change. Curious that the example plugin still seems to work. Passing the arrays here may not function as expected.

Comment (msamadony) [reply app/packages/operators/src/state.ts:136]: Yeah, I also wonder why. Probably one of the other dependencies gets updated too.

**Pull request merged**